brainstormforce/astra
# AST-2532 - fix: Spectra container spacing options not working

### Description

- Spectra container spacing options 'padding' & 'margin' do not work when astra is activated.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2532
### Screenshots

https://d.pr/v/ZqF9fI
### Types of changes


bugfix: Spectra container spacing options 'padding' & 'margin' do not work when astra is activated.



### How has this been tested?

Step to reproduce:

- Add a container or any other block 
- Go to that container style section
- Add padding and margin for tablet and mobile
- Check on frontend 
- Margin right and left is not working for all devices

### Checklist:
